src/routes/categories/components/__tests__/__snapshots__/CategoryTimeStamp.spec.js.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CategoryTimeStamp should match snapshot 1`] = `
+<CategoryTimeStampstyles__TimeStamp>
+  lastUpdate
+  18. November 2017
```

Comment:
Isn't this platform specific? I think its possible that this test fails on other platforms with an other locale.

Reply:
What do you mean by platform specific? The locale is the current language, so in the test it should always be the same.

Reply:
But the current language which moment uses is different on different platforms. To fix this I suggest we set the locale of moment in the setupTest.js.

I think the test would fail now if your system language would be arabic.

Reply:
No, it doesn't. The language/locale to which the timestamp is formatted is the language prop of CategoryTimeStamp, which is set in the test to 'de'. There is no way this could be different on different platforms.